db/migrate/20190416190949_initialize_state_for_users.rb

```diff
@@ -0,0 +1,18 @@
+class InitializeStateForUsers < ActiveRecord::Migration[5.2]
+  def up
+    User.find_each do |user|
+      user.activated! if user.accept_terms_date_utc.present?
+    end
+  end
+
+  def down
+    User.find_each do |user|
+      next unless user.active?
+
+      user.update(
+        accept_terms_date_utc: Time.current,
+        aasm_state:            'new'
+      )
+    end
+  end
```

Comment:
As @heidar pointed out elsewhere, here's a good opportunity to add `data_migrate` and `strong_migrations` gems, and to move this into a data migration.

Reply:
No, let's do that as part of a separate PR